crates/orchestrator/src/config.rs

```diff
@@ -161,3 +175,10 @@ async fn build_settlement_client(settings_provider: &impl SettingsProvider) -> B
         _ => panic!("Unsupported Settlement layer"),
     }
 }
+
+async fn build_storage_client() -> Box<dyn DataStorage + Send + Sync> {
+    match get_env_var_or_panic("DATA_STORAGE").as_str() {
+        "aws" => Box::new(AWSS3::new(AWSS3Config::new_from_env()).await),
```

Comment:
should we make this s3? aws is not self explanatory

Reply:
done